poap-xyz/poap-gallery
# 🐛 Fix: Use static json provider

## Summary

Use StaticJsonRpcProvider to avoid calling `eth_chainId`. We only call mainnet for ENS.

We are doing unnecessary calls to `eth_chainId`

![Screenshot from 2021-09-27 18-29-49](https://user-images.githubusercontent.com/19600590/134988096-768677dc-b94e-495e-9d26-ea3758e1ce17.png)

https://docs.ethers.io/v5/api/providers/jsonrpc-provider/#StaticJsonRpcProvider

Gallery now:
You can see there are two calls. The first one is with `eth_chainId`
![Screenshot from 2021-09-27 18-35-09](https://user-images.githubusercontent.com/19600590/134988813-ff9922bf-a73b-4c67-ae23-18261eaf131e.png)


With the changes:
There is only one call with `eth_call`


## Type of chang
![Screenshot from 2021-09-27 18-37-31](https://user-images.githubusercontent.com/19600590/134989008-36bbc29a-6820-4b94-ad58-c13cfed7a883.png)
e


## Testing

- Verify that ENS is still working. 
- Verify that we don't call eth_chainId every time we are doind a eth_call 
## Ticket

**[Change!]** Add your issue/ticket (if applies)

## Dependencies

**[Change!]** Let us know if your PR has a dependency with another PR

## Checklist:
